ed448/src/decaf/points.rs

```diff
@@ -761,6 +759,34 @@ mod test {
         assert_ne!(point, DecafPoint::GENERATOR);
     }
 
+    #[test]
+    fn voprf_test() {
+        use elliptic_curve::hash2curve::ExpandMsgXof;
+        use hex_literal::hex;
+        use sha3::Shake256;
+
+        let blind = Scalar::from_bytes_mod_order(&ScalarBytes::from(hex!("64d37aed22a27f5191de1c1d69fadb899d8862b58eb4220029e036ec65fa3833a26e9388336361686ff1f83df55046504dfecad8549ba11200")));
+        // https://www.rfc-editor.org/rfc/rfc9497#:~:text=Input%20%3D%2000-,Blind%20%3D%2064d37aed22a27f5191de1c1d69fadb899d8862b58eb4220029e036ec65fa%0A3833a26e9388336361686ff1f83df55046504dfecad8549ba112,-BlindedElement%20%3D%207261bbc335c664ba788f1b1a1a4cd5190cc30e787ef277665ac%0A1d314f8861e3ec11854ce3ddd42035d9e0f5cddde324c332d8c880abc00eb
+        assert_eq!(blind.to_bytes(), hex!("64d37aed22a27f5191de1c1d69fadb899d8862b58eb4220029e036ec65fa3833a26e9388336361686ff1f83df55046504dfecad8549ba112"));
+
+        let mut uniform_bytes = [0; 112];
+        ExpandMsgXof::<Shake256>::expand_message(
+            &[&[0]],
+            &[b"HashToGroup-OPRFV1-\x01-decaf448-SHAKE256"],
+            112,
+        )
+        .unwrap()
+        .fill_bytes(&mut uniform_bytes);
+        assert_eq!(uniform_bytes, hex!("16a43ca9148330f7fd70d04fdeaeba3a0fdf30b9cbccf34875da4c5989080b5a8430ecc590fec29293965431c49822a5e1101849ff009e121b60178013f35572ef767075f3d10f51829a0dbe45f49a04aded911cee38b85499940ae8938a3a30c8a7f99218ea67bad0ac0ace8535b974"));
+
+        let point = DecafPoint::from_uniform_bytes(&uniform_bytes);
+        assert_eq!(point.compress().as_bytes(), hex!("169f8fa3f4f97ae4b44bf595571e88d11e0ea2cce3ec0172f27e2a011709249c58722db5b9ecc010251dec7d8911f03f27c488b6a9579ddd"));
```

Comment:
I got these vectors from running the [POC from the official specification](https://github.com/cfrg/draft-irtf-cfrg-voprf/blob/43da59f797c92248c0362e4c74cc10a8bd2b7b7d/poc/test_oprf.sage#L77).

Specifically, it starts to deviate when using `DecafPoint::from_uniform_bytes` uses `FieldElement::map_to_curve_decaf448()` ([`decaf448_map`](https://www.rfc-editor.org/rfc/rfc9496.html#name-element-derivation-2)).